testing/smoke-ide-test/build.gradle.kts

```
@@ -17,6 +15,17 @@ val smokeIdeTestSourceSet = sourceSets.create("smokeIdeTest") {
 addDependenciesAndConfigurations("smokeIde")
```

💅 The code could use an empty line here for better flow
```suggestion
addDependenciesAndConfigurations("smokeIde")

```